scraper.go

```diff
@@ -75,6 +75,7 @@ var missing = flag.String("missing", "", "The `file` where information about ROM
 var overviewLen = flag.Int("overview_len", 0, "If set it will truncate the overview of roms to `N` characters + ellipsis.")
 var ssUser = flag.String("ss_user", "", "The `username` for registered ScreenScraper users.")
 var ssPassword = flag.String("ss_password", "", "The `password` for registered ScreenScraper users.")
+var gdbAPIkey = flag.String("gdb_apikey", "", "The gamesdb apikey received by https://forums.thegamesdb.net/viewforum.php?f=10")
```

Comment:
Is this key per user or per app? looks like the key is for the dev then IP/key is used to determine quota? If so we may want to hardcode the key.

Reply:
Correct, its per IP, not sure if IP restrictions get carried over to other apps. I don't think they do, but would have to verify.

I didn't hardcode a key because I would rather you control the key that is used by registering for one and then adding it : )

Reply:
Oh yeah I already have one:
`fdb3e318c535c5f9fb5380d15cd6dbb5363cb197c1da897c7a268695658ceceb`

Feel free to add it as the default or as a fallback if it isn't provided.

Reply:
Added